.github/workflows/node.js.yml

```diff
@@ -0,0 +1,27 @@
+name: Node.js CI
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
```

Comment:
I know this is merged, but just going through issues now.  This should probably include 13 (and maybe 14? which releases tomorrow).

Reply:
oops yeah, sorry i'm in work mode where we only use LTS haha

Reply:
i realized when i did this that path-to-regexp went through like 5 major versions so I'll update this when I update path-to-regexp

Reply:
...or did path-to-regexp make this module useless? i forgot what I was thinking

Reply:
Wasn't the intent that this thin wrapper would replace direct usage of `path-to-regexp`?  I don't really know since I was not around at the time and it didn't seem to move forward.

Reply:
Somewhat, though it couldn't actually be used since this module included a higher `path-to-regexp` than we could use at the time.

We could still do that, but need to evaluate the parts of the logic that has changed; see if path-to-regexp itself cannot just do it now, etc.

I didn't want to _downgrade_ path-to-regexp on this module in order to make that happen, as folks are using it. I was thinking perhaps it could be revisited when router upped it's path-to-regexp version, which it has done now. The plan likely just need to be revisited to understand if it's even desirable and if so, what work do we need to do.

Reply:
https://github.com/pillarjs/path-to-regexp/releases/tag/v3.2.0 a match function was added to path-to-regexp 6 months ago, so I think this module is obsolete. i think I prefer it including `match` now instead of having 10000000 tiny modules since they're related anyways

Reply:
@blakeembrey thoughts?

Reply:
@jonathanong Yeah, you're correct. I made this package mostly obsolete and bundled it all into `path-to-regexp`.

Reply:
@blakeembrey sounds good 👍